### PR TITLE
Sogenactif bundle

### PR DESCRIPTION
**Pull request by ngdo-pro**
Hi team,

Please, remember to : 

- composer install (new parameters are created, you will need to adapt them to your project)
- chmod 755 src/SogenactifBundle/Api/bin/* (request and response files from Sogenactif Api)
- change absolute paths in /src/Sogenactif/Api/params/pathfile to fit your project 
- app/console doctrine:migrations:migrate (you may need to reset your database)
- app/console doctrine:fixture:load

To test it : 
- Login as user
- Search 
- Select a baptism
- Confirm
- Click on a payment mean
- Enter a fake card number like 1234567812345600 ( it has to end with 00 to be a valid payment, 05 to fail )
- Return to website with api button (Retour à la boutique)

Autoresponse does work online, can't be tested offline

### CODE REVIEW

File: src/AppBundle/Repository/BaptismHasUserRepository.php [hunk 1]
```diff
@@ -10,4 +12,20 @@
  */
 class BaptismHasUserRepository extends \Doctrine\ORM\EntityRepository
 {
+    /**
```
Comment (wafaworks) [inline src/AppBundle/Repository/BaptismHasUserRepository.php:15]: Sorry but the goal of this function doesn't appear very clear to me.. Could you be more precise on the commentary (maybe put an example?)

Comment (wafaworks) [reply src/AppBundle/Repository/BaptismHasUserRepository.php:15]: And it appears that you did the "find out" mistake in many files, it's "this function find**s** out". But that's not a real problem, just to be meticulous.. ;)

**Review by wafaworks — CHANGES_REQUESTED** [review]

File: src/SogenactifBundle/Resources/config/doctrine/Transaction.orm.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            nullable: true
+        captureDay:
+            type: string
+            length: 255
+            nullable: true
+        captureMode:
+            type: string
+            length: 255
+            nullable: true
+        cardNumber:
```
Comment (shog189) [inline src/SogenactifBundle/Resources/config/doctrine/Transaction.orm.yml:34]: About the entries relating to numbers, I do understand that it is a character chain but should it stay this way? Or couldn't it be a chain of INT?

**Review by Kaji-U — APPROVED** [review]
Good Job !

**Review by dbeauseigneur — APPROVED** [review]
OK

**Review by 2binfree — APPROVED** [review]
ok

**Review by wafaworks — APPROVED** [review]
Now it's all good for me, thanks for the comment changes. 👍

**Pull request merged**